ci-operator/step-registry/optional-operators/cvp-common/apply-secrets/optional-operators-cvp-common-apply-secrets-commands.sh

```diff
@@ -61,3 +61,38 @@ oc apply -f /tmp/kube_objects.yaml -n "$OO_INSTALL_NAMESPACE"
 
 # Remove the kube objects file just in case
 rm -rf /tmp/kube_objects.yaml
+
+IS_CUSTOM_KUBEOBJECTS_EXISTS=false
+
+echo "Check if the kube objects exists on the vault or not"
+
+KEYWORD_CUSTOM_KUBEOBJECTS="custom-"
+
+CUSTOM_KUBEOBJECTS_PATH=${CUSTOM_KUBEOBJECTS_PATH:=/var/run/}
+
+# the following command lists all the kubeobjects mounted on CUSTOM_KUBEOBJECTS_PATH 
+# with prefix "custom-" and also package name within the name of the file.
+
+LIST_OF_KUBEOBJECTS=$(ls "${CUSTOM_KUBEOBJECTS_PATH}""${KEYWORD_CUSTOM_KUBEOBJECTS}""${OO_PACKAGE}"*)
```

Comment:
Hi @samvarankashyap please keep in mind that not everything in this folder can be "oc-applied".
Currently, we work only with the file `kube_objects`. We could also consider defining a prefix `kube_objects_` and do some sorting if there are files `kube_objects_01`, `kube_objects_02`, ..., `kube_objects_99`